Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -2619,6 +2624,41 @@ private SpecialProcessKind GetProcessKind(AbsolutePath processOverride)
             return s_specialTools.TryGetValue(toolName.ToLowerInvariant(), out SpecialProcessKind kind) ? kind : SpecialProcessKind.NotSpecial;
         }
 
+        private static bool StringLooksLikeBuildXLTestTempFile(string fileName)

```

Comment:
>StringLooksLikeBuildXLTestTempFile [](start = 28, length = 34)

why is this addition necessary?

Reply:
There's no unit test for this special exception. I just want to add one to exercise the feature.

---
In reply to: [314459941](https://github.com/microsoft/BuildXL/pull/741#discussion_r314459941) [](ancestors = 314459941)

Reply:
I'm not following. This adds a new exception rule for a new process into the product code. Why does adding a unit test necessitate that?

---
In reply to: [314472403](https://github.com/microsoft/BuildXL/pull/741#discussion_r314472403) [](ancestors = 314472403,314459941)